package.json

```diff
@@ -13,8 +13,11 @@
   "scripts": {
     "start": "node index",
     "build": "grunt && handlebars client/views/ -e tpl -f client/js/lounge.templates.js",
-    "test": "HOME=test/fixtures mocha test/**/*.js && npm run lint",
-    "lint": "eslint . && stylelint \"**/*.css\"",
+    "test": "HOME=test/fixtures mocha test/**/*.js",
```

Comment:
Is it possible to keep `HOME=test/fixtures` in the according test to keep [this](https://github.com/thelounge/lounge/pull/260#discussion-diff-59969310)?